LearningMissionLab/Schedule.cs

```diff
@@ -17,11 +17,16 @@ public class Schedule
             this._startDate = startDate;
             this._finishDate = finishDate;
             this._scheduleEventList = scheduleEventList;
-            this._createDate = DateTime.UtcNow;
-            this._updateDate = DateTime.UtcNow;
+            this._createDate = createDate;
+            this._updateDate = updateDate;
+        }
+        public Schedule(DateTime startDate, DateTime finishDate, List<ScheduleEvent> scheduleEventList)
+        {
+            this._startDate = startDate;
+            this._finishDate = finishDate;
+            this._scheduleEventList = scheduleEventList;
```

Comment:
Add:
  this._createDate = DateTime.UtcNow;
  this._updateDate = DateTime.UtcNow;